components/contribute-cards/ContributeCrypto.js

```diff
@@ -20,7 +20,7 @@ const messages = defineMessages({
 const ContributeCrypto = ({ intl, collective, contributors, stats, ...props }) => {
   return (
     <Contribute
-      route={`/${collective.slug}/contribute/crypto`}
+      route={`/${collective.slug}/payment/crypto`}
```

Comment:
Why introduce this `payment` suddenly?

Reply:
@znarf : Here my thinking was to introduce a verb other than `contribute` as we cannot use `contribute` as explained above; https://github.com/opencollective/opencollective-frontend/pull/6689#discussion_r684470100. I thought that since crypto is a payment method, we add the verb `payment`. Maybe it's not the best; do you any alternative suggestions? 🤔

Reply:
`payment` is not a verb.

Reply:
If `contribute` is not an option, why not use `donate`?

Reply:
Done. I've changed this to `donate/crypto` instead. I think this should be okay. 🤔